zbsprimarystorage/zbsprimarystorage/zbsagent.py

```
@@ -383,7 +384,7 @@ def clone_volume(self, req):
         if not ret.result.hasattr('fileInfo'):
             raise Exception('failed to found snapshot for volume[%s]' % cmd.lunName)
         for info in ret.result.fileInfo:
-            if cmd.snapshotName in info.fileName:
+            if snapshot in info.fileName:
```

Ruff (0.12.2)

`snapshot` may be undefined, or defined from star imports

(F405)